compiler/quilt/tools/command.py

```diff
@@ -148,21 +153,33 @@ def get_registry_url(team):
     _registry_url = url or DEFAULT_REGISTRY_URL
     return _registry_url
 
-def config():
-    answer = input("Please enter the URL for your custom Quilt registry (ask your administrator),\n"
-                   "or leave this line blank to use the default registry: ")
+def config(team=None):
+    if team is None:
+        message = "Please enter the URL for your custom Quilt registry " \
+                  "(ask your administrator),\n" \
+                  "or leave this line blank to use the default registry: "
+    else:
+        message = "Please enter the URL for the Quilt Team registry [%s] " \
+                  "(ask your administrator),\n" \
+                  "or leave this line blank to use the default: " % team
+    answer = input(message)
+
+    # When saving the config, store '' instead of the actual URL in case we ever change it.
+    cfg = _load_config()
     if answer:
         url = urlparse(answer.rstrip('/'))
         if (url.scheme not in ['http', 'https'] or not url.netloc or
             url.path or url.params or url.query or url.fragment):
             raise CommandException("Invalid URL: %s" % answer)
         canonical_url = urlunparse(url)
-    else:
-        # When saving the config, store '' instead of the actual URL in case we ever change it.
-        canonical_url = ''
-
-    cfg = _load_config()
-    cfg['registry_url'] = canonical_url
+        
+        if team:
+            _check_team_id(team)
+            cfg['team_registry_url'] = canonical_url
+            cfg['team_id'] = team
+        else:
+            cfg['registry_url'] = canonical_url
+            cfg['team_id'] = ''
```

Comment:
Should this be `None`?

Reply:
Ok. Doesn't really matter, I guess... Just make sure `config('')` doesn't do something bad. The two different checks, `if team` vs `if team is not None`, look like they could cause problems.